### ISSUE TITLE
Followup on: "Dedicated integrated medals area #7" Showing what medal you currently have

### ISSUE THREAD

**Issue by st-AR-gazer** [MisfitMaid/tm-map-info#15]
I've been using this feature for a while now, and I've noticed that it's really cumbersome to check what time you currently have when only using this plugin. Would it be possible to add a way to perhaps darken/lighten/change the background of the specific medal you currently have?
![image](https://github.com/MisfitMaid/tm-map-info/assets/132480072/83991d1b-e139-4efb-b1e0-628b6848f7a6)
Maybe adding a tint/hue to the background instead of just adjusting the brightness would work better?
:)

**Comment by XertroV** [comment]
yep for sure. I'm not sure about this exact idea, might have a think about alternative ways to show PB (so that it's a more complete replacement for common Ultimate Medals cases, which I guess is the inspiration for this issue)

**Comment by XertroV** [comment]
Like maybe an option to just show your PB along with the other times might be nicer. Then you have the visual indication of medal as well as more direct feedback

**Comment by st-AR-gazer** [comment]
Well, that was fast. Response time was 10/10. Anyway, the goal for me would be to be able to look at this out of the corner of my eye and see the current medal time.
Something like this would, in my opinion, be optimal to achieve that (ignore the oversaturation, xdd).
![image](https://github.com/MisfitMaid/tm-map-info/assets/132480072/60008218-f5a7-4fe6-9d88-b68682a80f43)
And something like this for AT, I guess (again, ignore the saturation, xdd):
![image](https://github.com/MisfitMaid/tm-map-info/assets/132480072/af36b01d-40ad-4bf7-996b-b9a2bcad3e70)

**Comment by XertroV** [comment]
with the current patch there's an option to show PB deltas like the below. Both the positive and negative numbers are configurable. That can achieve the same end, does that work?

![image](https://github.com/MisfitMaid/tm-map-info/assets/1046448/f6672217-1ca4-4870-8dbe-908ff78aa261)

**Comment by XertroV** [comment]
example of gold medal, with 'Hide Medals after the first worst than PB' option on and off

![image](https://github.com/MisfitMaid/tm-map-info/assets/1046448/3bcd0650-a673-41e4-8f8b-43e8fa2db8d1)
![image](https://github.com/MisfitMaid/tm-map-info/assets/1046448/48d55bc6-9db1-4f18-965f-48475030bda9)

**Comment by st-AR-gazer** [comment]
Love it ++

**Comment by Fort-TM** [comment]
Sounds like this is fixed 👍
